### PR TITLE
fix a potential exception in loading culture info

### PR DESCRIPTION
**Pull request by SaschaKP**
avoid using a non-existent culture, use current culture available LCID and set proper decimal and thousand separator chars

### CODE REVIEW

File: Steam/Interpreter.cs [hunk 1]
```diff
@@ -718,7 +718,7 @@ public static class Interpreter
 
         static Interpreter()
         {
-            Culture = new CultureInfo("en-EN", false);
+            Culture = new CultureInfo(CultureInfo.CurrentCulture.LCID, false);
```
Comment (andreakarasho) [inline Steam/Interpreter.cs:721]: Why not Invariant?

Comment (SaschaKP) [reply Steam/Interpreter.cs]: it's of course a performance reason...

image

Comment (SaschaKP) [reply Steam/Interpreter.cs:721]: normally I would say, as you've been notified, that it's a performance reason, but making some benchmarks to be absolutely sure, in try parse from strings, it doesn't make really any difference, it only makes a difference (and local culture is more performant) when using .ToLower, to upper etc.

**Comment by jaedan** [comment]
Merged fix. Thanks!

**Pull request closed without merging**